### PR TITLE
Adip membership

### PR DESCRIPTION
**Pull request by pmuir**
Deck completed to date

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -13,6 +13,14 @@ export const routes: Routes = [
     pathMatch: 'full'
   },
   {
+    path: 'home',
```
Comment (joshuawilson) [inline src/app/app-routing.module.ts]: @pmuir why did you add a Home route? The first route takes you to Home. If you want a direct path then you should use the Login example. The style you used here is for lazy loading.

File: src/app/home/home-routing.module.ts [hunk 2]
```diff
@@ -6,11 +6,6 @@ import { HomeComponent } from './home.component';
 const routes: Routes = [
   {
     path: '',
-    redirectTo: 'home',
```
Comment (joshuawilson) [inline src/app/home/home-routing.module.ts]: This is how you do it for the default page. Is Home not the default anymore?

File: src/app/profile/profile-routing.module.ts [hunk 3]
```diff
@@ -15,8 +15,10 @@ const routes: Routes = [
     component: ProfileComponent,
     children: [
       { path: '',          component: OverviewComponent },
-      { path: 'spaces',    loadChildren: './collaboration/collaboration.module#CollaborationModule' },
+      { path: 'spaces',    loadChildren: './spaces/spaces.module#SpacesModule' },
       { path: 'resources', loadChildren: './resources/resources.module#ResourcesModule' },
+      { path: '', loadChildren: './resources/resources.module#ResourcesModule' }
```
Comment (joshuawilson) [inline src/app/profile/profile-routing.module.ts]: What are you trying to do here without the path? The first line with no path is the default going to the Overview page.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (pmuir) [inline src/app/app-routing.module.ts]: Hmm, I seem to have not actually comitted all the changes to app-routing - as routes for public, signin and others are missing. I will have to check in the morning, as I've not got the laptop I was using on me right now.

To answer your question - home is no longer the "first route" as you are calling it, public is, as it's the landing page for the app right now - this allows us to present a logged in and not logged in experience - but as those changes are missing...

File: src/app/home/home-routing.module.ts (hunk 2 above)
Comment (pmuir) [inline src/app/home/home-routing.module.ts]: No, it's not, public should be, but I must have missed some changes.

File: src/app/profile/profile-routing.module.ts (hunk 3 above)
Comment (pmuir) [inline src/app/profile/profile-routing.module.ts]: Will fix.
